### ISSUE TITLE
unable to input or select VET-USD 

### ISSUE THREAD

**Issue by PtrckM** [CoinAlpha/hummingbot#2790]
**Describe the bug**
- Unable to input or select `VET-USD` 

**Steps To Reproduce**
1. connect binance_us
2. create PMM as strategy
3. input `VET-USD` will get notified `VET-USD is not an active market on binance_us.` while on [binance.us](https://www.binance.us/en/trade/VET_USD) exchange its available.

```
Enter the token trading pair you would like to trade on binance_us (e.g. BTC-USDT) >>> VET-USD
VET-USD is not an active market on binance_us.
```

**Screenshots**
![image](https://user-images.githubusercontent.com/54516858/103451640-89f87680-4d01-11eb-9218-e2d93bed2e70.png)

- As a workaround select `VE-TUSD` after typing `VE` then press `tab` for autocomplete
![image](https://user-images.githubusercontent.com/54516858/103451666-c4faaa00-4d01-11eb-89a2-1cf77e1e0b7d.png)

- To confirm 
![image](https://user-images.githubusercontent.com/54516858/103451739-85808d80-4d02-11eb-80ea-726e788179b5.png)
![image](https://user-images.githubusercontent.com/54516858/103451741-8b766e80-4d02-11eb-9231-4edc165aeba8.png)

**Release version**
0.34.0

**Attachments**
- n/a

**Comment by PtrckM** [comment]
- Another similar pair `HNT-USD` bot doesn't allow to input or select as seen below

![image](https://user-images.githubusercontent.com/54516858/104797992-c9dd5480-57fd-11eb-97d6-ffded2f0636d.png)

- select `HN-TUSD` for trading pair `HNT-USD` confirm pair orderbook and ticker
![image](https://user-images.githubusercontent.com/54516858/104798025-28a2ce00-57fe-11eb-8144-9c6f8c84d460.png)
![image](https://user-images.githubusercontent.com/54516858/104798026-2cceeb80-57fe-11eb-897d-803b5dd09d9f.png)

**Comment by aarmoa** [comment]
I have created a PR with a possible fix to the issue: https://github.com/CoinAlpha/hummingbot/pull/3332

**Comment by dennisocana** [comment]
Fixed in version 0.39 release
